### ISSUE TITLE
[Screen reader - Logic apps - Design]: Screen reader does not announce any information when focus moves to the "Insert a new step after Create Job" button.

### ISSUE THREAD

**Issue by takyyon** [Azure/LogicAppsUX#6094]
### Describe the issue 

User Experience: 

 Visually impaired people who depend on screen reader users will not get information about the control due to which the user will not be able to understand the purpose of it and cannot perform any action. 

 Note: User credentials should NOT be included in the bug.

Pre-requisites:

- Make sure dependencies such as Azure Logic Apps (Standard), Azure Functions, C# are installed. 
- Navigate to Azure control present on the right side and select it. 
- Navigate to LogicApp icon present under workspace section, click on tat and select 'Create New Project'. 
- Navigate select stateful workflow, then enter the name of the workflow. 
- Once the workflow has been created then enter the name of the created workflow in edit field present on the top and press Enter. 
- Navigate to 'Open in current window' and select it. 
- A dialog will appear, select 'Yes', now the workflow will open in new window. 

Repro Steps:

- Open Visual Studio app. 
- Navigate to any workflow and right click on it, then select 'open designer'. 
- Navigate all the controls of the designer screen. 
- Observe if screen reader does not announce any information when focus moves to the "Insert a new step after Create Job" button. 

Actual Result: 

 Screen reader remains silent and does not announce any information when focus moves to the "Insert a new step after Create Job" button.

 Observation: When multiple actions are added, screen reader does not announce any information when focus moves to the "Insert a new step after Create Job" buttons. It only announces the information of "Insert a new step after Create Job" button of last action.

Expected result: 

 Screen reader should not remain silent and should announce information of all the "Insert a new step after Create Job" buttons when focus moves to them.

### Are you using new designer or old designer 

New Designer (With action panels)

### Screenshots or Videos 

*No response*

### Browser 

Chrome, Edge, Safari

### Additional context 

*No response*

**Comment by ccastrotrejo** [comment]
Tested this issue in vscode and standalone (chrome, edge & safari) and with different screen readers. This is fixed by now
